vta/hardware/chisel/src/main/scala/core/TensorGemm.scala

```diff
@@ -180,10 +183,10 @@ class TensorGemm(debug: Boolean = false)(implicit p: Parameters) extends Module
   val done = inflight === 0.U &
              ((state === sExe &
               cnt_o === dec.lp_0 - 1.U &
-              cnt_i === dec.lp_1 - 1.U &
-              uop_idx === uop_end - 1.U &
-              inflight === 0.U) |
-             state === sWait)
+	      cnt_i === dec.lp_1 - 1.U &
```

Comment:
Are you inserting tabs instead of spaces? can you search for `\t` in your editor and replace them with spaces accordingly?

Reply:
fixed